lib/vehicle.rb

```diff
@@ -22,4 +27,16 @@ def antique?
   def electric_vehicle?
     @engine == :ev
   end
+
+
+  def set_plate_type_dawggy
+    date = Time.new.strftime("%Y")
+    return @plate_type = :antique if (date.to_i - @year) >= 25
+    return @plate_type = :ev if @engine == :ev
+    return @plate_type = :regular
```

Comment:
You're guard clause happy now! lol. One guard clause, if you need more, rewrite it into an if block. It's much mor readable and allows people to see the flow of data better.